third_party/minio/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN sh /third_party/download_source.sh </third_party/minio/repo-MPL.txt
 
 
 # Minio image
-FROM minio/minio:RELEASE.2019-08-14T20-37-41Z
+FROM minio/minio:RELEASE.2020-12-18T03-27-42Z
```

Comment:
is this required for this PR?

Reply:
@HumairAK the image used here is published few years back. We enabled the s390x and ppc64le builds later after this image being build. Hence we are requesting to update this. You can see from tag, the image used is released in 2019 and we are having image supported from 2020.